queries.sql

```diff
@@ -1,3 +1,27 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name ~~ '%mon'
+
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-1-1' AND '2019-12-31'
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu'
+
+-- List name and escape attempts of animals that weigh more than 10.5kg
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+-- Find all animals that are neutered.
+
+SELECT * FROM animals WHERE neutered = false
```

Comment:
**### - [ ] optional**

- [ ] Hello and nice job but please update the `SELECT * FROM animals WHERE neutered = false` to true